xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/commentsinline.vm

```diff
@@ -292,7 +294,9 @@ $xwiki.ssfx.use('uicomponents/viewers/comments.css', true)
 #if($date), <span class="commentdate">$!xwiki.formatDate($date)</span>#end
       </div>
     </div>## commentheader
-    <div class="commentcontent">$doc.display('comment', 'edit', $comment)</div>
+    <div class="commentcontent wysiwyg-field-${comment.number}">
```

Comment:
Same comment as above: is it always a WYSIWYG field?

Reply:
ok I need to check back this one specifically. I think the field name is the same even with a simple text editor, but when I tested the javascript handles it properly so I guess Manuel perform the changes to handle this new field class. 
Might be a bit dangerous to change now...

Reply:
So I checked locally and I confirm we always obtain a wysiwyg-field whatever the editor is. This classname is currently used in comments.js, and in the pageobject. I guess we should be able to rename it for something more generic such as "comment-editor".

Reply:
I renamed it to "commenteditor".